dockerfile-github-actions/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Image for building async-profiler release packages on github actions
+
+# Stage 0: download musl sources and build toolchains
+FROM public.ecr.aws/lts/ubuntu:24.04
```

Comment:
I've chosen 18.04, since there were backward compatibility issues when building on newer distros.
Can you verify that:
1. Result binaries are compatible with GLIBC 2.14 on x86 and with GLIBC 2.17 on ARM64? I.e. there should be no symbols requiring newer GLIBC when running `nm build/lib/libasyncProfiler.so | grep GLIBC_2.[0-9][0-9]`.
2. Result binaries work on Alpine Linux.

Reply:
will check, if not will revert back to 18.04 and try building node20 from source

Reply:
I see some that need newer GLIBC `U pthread_mutexattr_init@GLIBC_2.34`